ansible/roles/tcpdump-moloch/files/enodump.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=enodump packet capture
+
+
+[Service]
+Restart=always
+TimeoutStartSec=300
+WorkingDirectory=/pcaps
+
+ExecStart=/usr/sbin/tcpdump -i vulnboxes --packet-buffered -G 60 -w './data/enoflag_%Y-%m-%d_%H_%M_%S.pcap' -s 0
```

Comment:
Naming the files `.pcap` as they are written to causes problems when working with rsync. rsync will try to sync files while they are being written to and tries to remove them afterwards (not sure what happens in that case). I am also not sure if moloch picks this up properly if the file is copied once and then overwritten (or appended to) later, in case rsync does not delete the files before they are completed.

Reply:
Any update here?